src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import static edu.wpi.first.wpilibj.DoubleSolenoid.Value.kForward;
+import static edu.wpi.first.wpilibj.DoubleSolenoid.Value.kReverse;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.IntakeConstants;
+
+// import edu.wpi.first.wpilibj.motorcontrol.PWMMotorController;
+// import edu.wpi.first.wpilibj.motorcontrol.PWMVictorSPX;
+
+public class Intake extends SubsystemBase {
+
+  private final DoubleSolenoid m_solenoidLeader =
+      new DoubleSolenoid(
+          PneumaticsModuleType.CTREPCM,
+          IntakeConstants.kLeaderForward,
+          IntakeConstants.kLeaderReverse);
+
+  private final DoubleSolenoid m_solenoidFollower =
+      new DoubleSolenoid(
+          PneumaticsModuleType.CTREPCM,
+          IntakeConstants.kFollowerForward,
+          IntakeConstants.kFollowerReverse);
```

Comment:
We should only need one solenoid so we can remove the follower and just have:
`m_intakeSolenoid` and also rename the constants